plutus-use-cases/src/Language/PlutusTx/Coordination/Contracts/CrowdFunding.hs

```diff
@@ -190,7 +187,8 @@ theCampaign = Campaign
 --   refund if the funding target was not met.
 contribute :: AsContractError e => Campaign -> Contract CrowdfundingSchema e ()
 contribute cmp = do
-    Contribution{contribValue, contributor} <- endpoint @"contribute"
+    Contribution{contribValue} <- endpoint @"contribute"
+    contributor <- ownPubKey
```

Comment:
I still kind of want to say that you might want this sort of thing to be more generic. But this is an example after all.